app/controllers/application_controller.rb

```diff
@@ -17,6 +17,10 @@ def current_admin?
     current_user && current_user.admin?
   end
 
+  def store_owner?
```

Comment:
This seems like a method that should be part of the User model rather than a helper method. Am I missing something? It just returns true if the current user is a store owner... but if it was on the user model, you could call it for any user... even though you'd probably only ever call it for the current user.